### PR TITLE
Enhancement: Revert to default logging

### PR DESCRIPTION
**Pull request by CM000n**
With this PR, our central logging config is rolled back. We as lib should never overwrite the root logger of the application that uses us.
This can cause problems, as all logs of the main application will be intercepted by our logger. This should fix this behaviour:  https://github.com/DurgNomis-drol/ha_toyota/issues/242

### CODE REVIEW

**Review by moonter45 — APPROVED** [review]
tested and working well :)

File: simple_client_example.py
Comment (DurgNomis-drol) [inline simple_client_example.py:13]: What if this is not imported through HA but used standalone? Don't we still need to invoke the logger somewhere? No nothing about it, so just a question

**Comment by fenty17** [comment]
Is this going to be merged soon? Just spent ages searching for the cause of my missing logs! Great work on the integration by the way.

**Comment by desosav** [comment]
> Is this going to be merged soon? Just spent ages searching for the cause of my missing logs! Great work on the integration by the way.

You can select the specific branch until it is merged by updating the custom_components/mytoyota/manifest.json file in homeassistant.
Replace requirements with the following and restart

` "requirements": [
    "mytoyota @ git+https://github.com/DurgNomis-drol/mytoyota/tree/enhancement/revert_to_default_logging",
    "arrow"
  ],`

I am so glad my error messages are back :)

File: simple_client_example.py
Comment (CM000n) [reply simple_client_example.py:13]: The logger must be initiated by the programme that uses the lib. In the case of Home Assistant, it is the root logger of Home Assistant.
If you use the lib directly, e.g. via `simple_client_example.py`, the logger must be initiated directly in it if required. But we should leave that to the user.

**Comment by CM000n** [comment]
> Is this going to be merged soon? Just spent ages searching for the cause of my missing logs! Great work on the integration by the way.

Still waiting for approval 😞

**Review by GitOldGrumpy — APPROVED** [review]

**Pull request merged**
